R/autoplot.R

```diff
@@ -0,0 +1,156 @@
+#' Automatically plot an epi_df
+#'
+#' @param object An `epi_df`
+#' @param ... <[`tidy-select`][dplyr_tidy_select]> One or more unquoted
+#'   expressions separated by commas. Variable names can be used as if they
+#'   were positions in the data frame, so expressions like `x:y` can
+#'   be used to select a range of variables.
+#' @param .color_by Which variables should determine the color(s) used to plot
+#'   lines. Options include:
+#'   * `all_keys` - the default uses the interaction of any key variables
+#'     including the `geo_value`
+#'   * `geo_value` - `geo_value` only
+#'   * `other_keys` - any available keys that are not `geo_value`
+#'   * `.response` - the numeric variables (same as the y-axis)
+#'   * `all` - uses the interaction of all keys and numeric variables
```

Comment:
issue: Getting an error using the "all" setting. I get the same error when trying to use `.facet_by = all`.

```
autoplot(aa, case_rate_7d_av, .color_by="all")
# Error in `dplyr::mutate()`:
# ℹ In argument: `.colours = switch(...)`.
# Caused by error in `interaction()`:
# ! object '.response_name' not found
```

Reply:
issue: `.response` and `none` generate the same plot, where the data is a single color. It sounds like `.response` is supposed to make higher values a different color than lower values.

Reply:
I added it as an example.